shared/wallets/confirm-form/participants/index.js

```diff
@@ -76,7 +76,7 @@ const Participants = (props: ParticipantsProps) => {
     <Kb.Box2 direction="vertical" fullWidth={true}>
       <ParticipantsRow heading="From">
         <AccountEntry
-          contents={props.fromAccountAssets}
```

Comment:
i guess a previous change already removed the '+ more'?

Reply:
yeah I only saw it in some storybook text